flink-cdc-common/src/main/java/com/ververica/cdc/common/event/AddColumnEvent.java

```diff
@@ -66,7 +66,7 @@ public static class ColumnWithPosition implements Serializable {
         private final ColumnPosition position;
 
         /** The added column lies in the position relative to this column. */
-        private final @Nullable Column existingColumn;
```

Comment:
Will `existingColumnName` be better to keep pace with `DropColumnEvent`?

Reply:
Done.